test/Core.Test/Services/PolicyServiceTests.cs

```diff
@@ -126,12 +126,16 @@ await sutProvider.GetDependency<IPolicyRepository>()
                 .UpsertAsync(default);
         }
 
-        [Theory, CustomAutoData(typeof(SutProviderCustomization))]
-        public async Task SaveAsync_SingleOrg_KeyConnectorEnabled_ThrowsBadRequest(
-            [PolicyFixtures.Policy(Enums.PolicyType.SingleOrg)] Core.Models.Table.Policy policy,
+        [Theory]
+        [InlineCustomAutoData(new[] { typeof(SutProviderCustomization) }, Enums.PolicyType.SingleOrg)]
+        [InlineCustomAutoData(new[] { typeof(SutProviderCustomization) }, Enums.PolicyType.RequireSso)]
+        public async Task SaveAsync_PolicyRequiredByKeyConnector_DisablePolicy_ThrowsBadRequest(
```

Comment:
Just an optional suggestion: I know it's pretty long, but I find it easier to read and understand what is being tested. 
```suggestion
        public async Task SaveAsync_Throws_BadRequest_When_Disabling_A_Given_Policy_Which_The_KeyConnector_Requires(
```
The other naming-pattern is used across other tests and makes sense to keep it consistant. Maybe it's just something I need to get used to.

Reply:
@djsmith85 I agree with your suggestion but let's discuss it in #code before we change established patterns.